### ISSUE TITLE
Minimize animation  broken after update

### ISSUE THREAD

**Issue by F3R96** [mydockfinder/mydockfinder#659]
Hi! Like the title says, the minimize animation stops working after a few hours, and I end rebooting the app, this didn´t happen before.
Any chance of rolling back to preview version? It was working amazing two releases ago

**Comment by mydockfinder** [comment]
The beta channel has 1.9.9.6




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年2月26日(星期一) 上午7:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [mydockfinder/mydockfinder-for-Win10-Win11] Minimize animation  broken after update (Issue #659)





 
Hi! Like the title says, the minimize animation stops working after a few hours, and I end rebooting the app, this didn´t happen before.
 Any chance of rolling back to preview version? It was working amazing two releases ago

**Comment by F3R96** [comment]
> The beta channel has 1.9.9.6
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: ***@***.***&gt;; 发送时间: 2024年2月26日(星期一) 上午7:44 收件人: ***@***.***&gt;; 抄送: ***@***.***&gt;; 主题: [mydockfinder/mydockfinder-for-Win10-Win11] Minimize animation broken after update (Issue #659) Hi! Like the title says, the minimize animation stops working after a few hours, and I end rebooting the app, this didn´t happen before. Any chance of rolling back to preview version? It was working amazing two releases ago — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

Yes, but that one has a lot of bugs for me. That's why I didn't update mine until 1.9.9.7 came along. I wanna  use that same 1.9.9.7 that worked before

**Comment by mydockfinder** [comment]
There will be an update tonight and these issues will be fixed
&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年2月26日(星期一) 上午9:07
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Minimize animation  broken after update (Issue #659)





  
The beta channel has 1.9.9.6
 …
 ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: @.&gt;; 发送时间: 2024年2月26日(星期一) 上午7:44 收件人: @.&gt;; 抄送: @.&gt;; 主题: [mydockfinder/mydockfinder-for-Win10-Win11] Minimize animation broken after update (Issue #659) Hi! Like the title says, the minimize animation stops working after a few hours, and I end rebooting the app, this didn´t happen before. Any chance of rolling back to preview version? It was working amazing two releases ago — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: @.&gt;
  
Yes, but that one has a lot of bugs for me. That's why I didn't update mine until 1.9.9.7 came along. I wanna  use that same 1.9.9.7 that worked before

**Comment by F3R96** [comment]
I tested the update, and still having the same issue. And also the app starts to ignore all the new apps i open and they wont appear on the dock after a while

**Comment by mydockfinder** [comment]
Can you record a video, I'll see how it is

**Comment by F3R96** [comment]
Sadly, I can't do it right now. I manage to get the version I had before and it works. 

If you want to replicate the same bug, you just have to use the app for an hour and then start to open some apps. They will not show on the dock and instead of showing the name you assing to it, it will show the name of the process (all this on the finder bar) 
For example: Discord will start to show as update on the finderbar even though I change the name.


Also, the minimize animation breaks and the apps will start to use the default one to minimize.

**Comment by F3R96** [comment]
Here's a video with the bugs I talk to you about:
https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/cceff9da-53a9-4f6b-b779-43e675350344

You can see discord it's not on the dock and it says "update" when previously I change it to "discord" Also OBS is missing on the dock.

At first I though this had something to do with me using two screens now, but I don't know...

**Comment by F3R96** [comment]
This is after restarting the app
https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/161c2a03-7ea0-4b6a-8745-99d573a6d379

**Comment by F3R96** [comment]
I can confirm that the issues are triggered by using dual screens

**Comment by mydockfinder** [comment]
What is the win10 version number of your system




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年2月28日(星期三) 上午7:44
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Minimize animation  broken after update (Issue #659)





 
I can confirm that the issues are triggered by using dual screens
 
&mdash;

**Comment by F3R96** [comment]
![imagen](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/0da74971-889f-4b0c-9aa4-33c65fbd0a7a)

**Comment by mydockfinder** [comment]
Try to turn on HDR,Will it be better to minimize the animation?,Because the UWP screenshot of the win10 system has been very poorly supported,Microsoft doesn't seem to plan to update this feature,So the UWP screenshot interface of the win10 system is very unstable

**Comment by mydockfinder** [comment]
Try to turn on HDR,Will it be better to minimize the animation?,Because the UWP screenshot of the win10 system has been very poorly supported,Microsoft doesn't seem to plan to update this feature,So the UWP screenshot interface of the win10 system is very unstable




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年2月28日(星期三) 中午11:18
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Minimize animation  broken after update (Issue #659)





 
imagen.png (view on web)
 
&mdash;

**Comment by F3R96** [comment]
https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/97b9a718-a5e3-4855-9fa5-c6b86e17c0b6

It happened again without using a second screen. MDF stops registering the apps I open after a while, don't know what to do.

Where can I find that hdr option? Maybe I already had it on

**Comment by mydockfinder** [comment]
Have you turned on the multi-screen display dock at the same time?




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年2月28日(星期三) 中午12:08
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Minimize animation  broken after update (Issue #659)





 
https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/78397863/97b9a718-a5e3-4855-9fa5-c6b86e17c0b6
 
It happened again without using a second screen. MDF stops registering the apps I open after a while, don't know what to do.
 
Where can I find that hdr option? Maybe I already had it on
 
&mdash;

**Comment by F3R96** [comment]
Yes, I had it on when using the dual screen. I also deactivated both finder and dock from the second screen and still having the same issue
